apps/web/src/constants.ts

```diff
@@ -18,19 +18,19 @@ const CHAIN_ID = {
   POLYGON: polygon.id,
 } as const;
 
-// This will be generated from Violet
-const CLIENT_ID =
-  "1212121212121212121212121212121212121212121212121212121212121214";
-
 // This will be generated from the ABI
 const TX_FUNCTION_SIGNATURE = "0x50d41df3";
 const TX_DATA =
   "0x000000000000000000000000d00f7eddc37631bc7bd9ebad8265b2785465a3b7000000000000000000000000000000000000000000000000000000001adc34a100000000000000000000000000000000000000000000000000000000000000e00000000000000000000000000000000000000000000000000000000000000000";
 const TX_CONTRACT_ADDRESS = "0xD2678cF600262057f485d12aD8F7c8FB5941EB46";
 
-const LOCAL_API_URL = "http://localhost:8080";
+// This is passed to the context
+const CLIENT_ID =
```

Comment:
Is this what gets used in prod?

Reply:
No, but you can change this on the Parameters